komodor/README.md

```diff
@@ -0,0 +1,34 @@
+# Komodor - Kubernetes Troubleshooting Platform
+
+## Overview
+
+This integration allows Datadog Monitor Alerts to be available in the Komodor Kubernetes troubleshooting platform and suggests related services based on service connections detected by Datadog.
+
+Komodor tracks changes across your entire K8s stack, analyzes their ripple effect, and provides you with the context you need to troubleshoot efficiently and independently. Komodor gives you insight into your Kubernetes deployments on a timeline with relevant information such as what changed, what code was pushed and by who. You can also view data from Git, config maps, your infrastructure, alerting and other tools such as Datadog, in one centralized and easy-to-understand display. 
+	
+With this integration, you can easily link to Datadog metrics with dynamic deployment links that take you directly to the matrices you need. This allows you to troubleshoot your microservices based on the most relevant context, connections and service dependencies detected in Datadog.
```

Comment:
Just wanted to confirm what's being referenced by `matrices` here -- would these be Datadog dashboards perhaps? Unless this term is well-known to Komodor users, it might be worthwhile to clarify this